odl/util/testutils.py

```diff
@@ -660,10 +660,17 @@ def test(arguments=None):
     pytest.main(args)
 
 
-def run_doctests(skip_if=False):
+def run_doctests(skip_if=False, extraglobs=None):
```

Comment:
I found this useful, but probably I should break it out from this pure contrib stuff.

Reply:
Agree, break it out.